GNUmakefile

```diff
@@ -422,6 +425,17 @@ cspc-operator-debug-image:
 	@cd buildscripts/${CSPC_OPERATOR_DEBUG} && sudo docker build -t ${HUB_USER}/${CSPC_OPERATOR_REPO_NAME}:inject --build-arg BUILD_DATE=${BUILD_DATE} .
 	@rm buildscripts/${CSPC_OPERATOR_DEBUG}/${CSPC_OPERATOR}
 
+.PHONY: cvc-operator-image
+cvc-operator-image:
+	@echo "----------------------------"
+	@echo -n "--> cvc-operator image "
+	@echo "${HUB_USER}/${CVC_OPERATOR_REPO_NAME}:${IMAGE_TAG}"
+	@echo "----------------------------"
+	@PNAME=${CVC_OPERATOR} CTLNAME=${CVC_OPERATOR} sh -c "'$(PWD)/buildscripts/build.sh'"
+	@cp bin/${CVC_OPERATOR}/${CVC_OPERATOR} buildscripts/cvc-operator/
```

Comment:
can we remove cvc-operator? and use ${CVC_OPERATOR}

Reply:
yes we can use , but i just follow the other tags